src/main/java/com/leinardi/pycharm/mypy/mpapi/MypyRunner.java

```diff
@@ -173,16 +181,8 @@ public static boolean checkMypyAvailable(Project project, boolean showNotificati
                 }
             }
         }
-        MypyConfigService mypyConfigService = MypyConfigService.getInstance(project);
-        if (mypyConfigService == null) {
-            throw new IllegalStateException("MypyConfigService is null");
-        }
-        String mypyPath = getMypyPath(project);
-        boolean isMypyPathValid = !mypyPath.isEmpty() && isMypyPathValid(mypyPath, project);
-        if (showNotifications && !isMypyPathValid) {
-            Notifications.showUnableToRunMypy(project);
-        }
-        return isMypyPathValid;
+
+        return false;
```

Comment:
Hey just to be sure: we can now remove the `return false;` at line 173 and 180 because we are now returning false in any case, right?